ansible/requirements.yml

```diff
@@ -193,3 +193,15 @@
 
 - src: ome.prometheus_postgres
   version: 0.3.0
+
+# https://github.com/ome/ansible-role-prometheus/pull/7/commits/fdcfaf93604b86fa753a2ae6d69d20d91c1292d1
+- name: ome.prometheus_new
```

Comment:
In the way of naming fun, I'd suggest something other than `_new`, `_new_2`, etc. since these can be left hanging around locally. Perhaps `ome.prometheus_${upcoming_version}-PR${PULL}`?